interfaces/cython/cantera/test/test_convert.py

```diff
@@ -81,6 +83,31 @@ def test_gri30(self):
         ref, gas = self.checkConversion('gri30.xml', 'gri30_test')
         self.checkKinetics(ref, gas, [300, 1500], [5e3, 1e5, 2e6])
 
+    def test_gri30_phases(self):
+
+        # cause all warnings to always be triggered.
+        warnings.simplefilter("always")
+
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
I think you can use `assertWarnsRegex()` here, right?

Reply:
correct. Didn't know this one (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertWarnsRegex)